### PR TITLE
Devops 1806 update infra images

### PR DESCRIPTION
**Pull request by jpravin**
Updating to the latest images + templating versions

### CODE REVIEW

File: configurator/sysdig-chart/templates/data-stores/redis-ha/redis-secondary-statefulset.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ spec:
       terminationGracePeriodSeconds: 10
       containers:
         - name: redis-secondary
-          image: {{ .Values.airgapped_registry_name | default "quay.io" }}/sysdig/redis:4.0.12.1-alpine-ha
+          image: {{ .Values.airgapped_registry_name | default "quay.io" }}/sysdig/redis:{{ .Values.sysdig.redisVersion }}-ha
```
Comment (mbreitung) [inline configurator/sysdig-chart/templates/data-stores/redis-ha/redis-secondary-statefulset.yaml]: I think that the full version should be replaced here

File: configurator/sysdig-chart/values.yaml [hunk 2]
```diff
@@ -39,8 +39,14 @@ sysdig:
...
   license: <change-me>
   dnsName: <change-me>
-  monitorVersion: 2271
+  monitorVersion: 2304
   anchoreVersion: v0.3.4.1
+  cassandraVersion: 2.1.21.13
+  elasticsearchVersion: 5.6.16.4
+  mysqlVersion: 5.6.44.0
+  postgresVersion: 10.6.10
+  redisVersion: 4.0.12.5
```
Comment (mbreitung) [inline configurator/sysdig-chart/values.yaml]: See comment above about using the full version here

File: configurator/sysdig-chart/templates/data-stores/redis-ha/redis-secondary-statefulset.yaml (hunk 1 above)
Comment (jpravin) [reply configurator/sysdig-chart/templates/data-stores/redis-ha/redis-secondary-statefulset.yaml]: Addressed. Added a new config in values.yaml for RedisHa.

File: configurator/sysdig-chart/tests/resources/large_aws/values.yaml [hunk 3]
```diff
@@ -37,6 +37,13 @@ sysdig:
   dnsName: foo.bar.com
   monitorVersion: 2304
   anchoreVersion: v0.3.4.1
+  cassandraVersion: 2.1.21.13
+  elasticsearchVersion: 5.6.16.4
```
Comment (janoe) [inline configurator/sysdig-chart/tests/resources/large_aws/values.yaml]: shouldn't be the elasticsearch version 5.6.16.5?
@bob-bins ?

File: configurator/sysdig-chart/tests/resources/large_aws_monitor_only/sysdig.json [hunk 4]
```diff
@@ -1253,7 +1253,7 @@
                 }
               }
             ],
-            "image": "quay.io/sysdig/elasticsearch:5.6.16.2",
+            "image": "quay.io/sysdig/elasticsearch:5.6.16.4",
```
Comment (janoe) [inline configurator/sysdig-chart/tests/resources/large_aws_monitor_only/sysdig.json]: Same here.
shouldn't be the elasticsearch version 5.6.16.5?
@bob-bins ?

**Pull request merged**